bundles/org.openhab.ui.habpanel/web/app/widgets/dummy/dummy.widget.js

```diff
@@ -47,7 +47,21 @@
                 vm.value = "N/A";
                 return;
             }
-            var value = item.transformedState || item.state;
+
+            if (vm.widget.usedescription) {
+                var options = item.stateDescription.options;
+                if (Array.isArray(options)) {
+                    var option = options.find((element) => element.value === item.state);
```

Comment:
This is probably the line that is causing problems (arrow functions are iOS Safari 10+, `find` is iOS Safari 8+).
The iPad 3 apparently runs on iOS 9.

Reference
https://caniuse.com/arrow-functions
https://caniuse.com/array-find
https://en.wikipedia.org/wiki/IPad_(3rd_generation)#Software

It could probably be chenged to `options.find((function (element) { return element.value === item.state; })` to make it compatible with iOS 8+.

Reply:
That would be great if it can be fixed as proposed.  I know this is old hardware, but it works great with HabPanel otherwise and HabPanel is a great legacy  alternative to the new MainUI that does require more modern hardware to run.  Your efforts are greatly appreciated.

Reply:
#1433 reimplements the code with techniques found elsewhere (and it seems to work in iOS 6).

![image](https://user-images.githubusercontent.com/2004147/175794044-8a9b7196-2176-4818-bba7-ca144e2c3a23.png)

Thank you for reporting that late minute bug.